galley/pkg/crd/validation/webhook.go

```diff
@@ -64,6 +64,8 @@ const (
 	watchDebounceDelay = 100 * time.Millisecond
 
 	httpsHandlerReadyPath = "/ready"
+
+	namespaceSelectorLabelKey = "istio-ns"
```

Comment:
What's istio-ns? Is this exposed to the user? There is no "Istio namespace" anymore, each component could be in different namespaces

Reply:
"istio-ns" is arbitrary, it's just a label key that can be applied to namespaces to mark what the webhook will use in the selector. 

e.g. let's say we have namespaces "istio-system-1", "istio-system-2", "bookinfo-1", and "bookinfo-2". We want updates to bookinfo-1 to use the istio-system-1 galley webhook. In this implementation, the customer would add "istio-ns: istio-system1" as a label to the bookinfo-1 namespace and add ```--enable-webhookNamespaceSelector=true``` to the command line in the galley deployment.

If there is a specific label key we should use (perhaps to be more consistent with other multi-tenant support?), please point me in the right direction. It could be as generic as "webhook-group" or "galley-webhook-group".

Reply:
In the Citadel namespace selection PR we used `ca.istio.io/env` for a label with this same behavior, I think it would be a good idea to scope the label to galley in that same way (i.e. use something like `galley.istio.io/env`). `istio-ns` is too broad and could have collisions with other component uses at some point.

 As much as possible we should keep the namespace targeting model consistent with what happens in the Citadel selection logic and the sidecar injector logic, inconsistency between these APIs wouldn't be ideal.

Reply:
That makes sense, we can use ```galley.istio.io/env```.